main.js

```diff
@@ -0,0 +1,109 @@
+var products = [];
+var seasonalSales = [];
+const discountSelect = document.getElementById("discountSelect");
+const productDisplayDiv = document.getElementById("productDisplayDiv");
+productDisplayDiv.innerHTML += `<div class="col-md-4"><div class="panel panel-primary"><div class="panel-heading"><h2 class="panel-title">Apparel</h2></div><div class="panel-body" id="apparel"></div></div></div>`;
+productDisplayDiv.innerHTML += `<div class="col-md-4"><div class="panel panel-primary"><div class="panel-heading"><h2 class="panel-title">Furniture</h2></div><div class="panel-body" id="furniture"></div></div></div>`;
+productDisplayDiv.innerHTML += `<div class="col-md-4"><div class="panel panel-primary"><div class="panel-heading"><h2 class="panel-title">Household</h2></div><div class="panel-body" id="household"></div></div></div>`;
+const apparelDiv = document.getElementById("apparel");
+const furnitureDiv = document.getElementById("furniture");
+const householdDiv = document.getElementById("household");
+var discountApplied = { //flags to determine whether discounts have already been triggered or not
```

Comment:
Where do discounts get reset when the season changes?  I don't see anywhere where you are clearing the discount.